Make.inc

```diff
@@ -1,4 +1,4 @@
-# -*- mode: makefile-gmake -*-
+# -*- mode: makefile -*-
```

Comment:
Thanks to Emacs, now we have proper syntax highlight in GitHub for this file :slightly_smiling_face: